File: push/src/main/java/com/xiaomi/mipush/sdk/stat/db/MessageDeleteJob.java
package com.xiaomi.mipush.sdk.stat.db;

import com.xiaomi.mipush.sdk.stat.db.base.DbManager;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/mipush/sdk/stat/db/MessageDeleteJob.class */
public class MessageDeleteJob extends DbManager.DeleteJob {
    protected String mDescription;

    public MessageDeleteJob(String str, String str2, String[] strArr, String str3) {
        super(str, str2, strArr);
        this.mDescription = "MessageDeleteJob";
        this.mDescription = str3;
    }

    public static MessageDeleteJob deleteUploadedJob(String str) {
        return new MessageDeleteJob(str, "status = ?", new String[]{String.valueOf(2)}, "a job build to delete uploaded job");
    }

    @Override // com.xiaomi.mipush.sdk.stat.db.base.DbManager.DeleteJob, com.xiaomi.mipush.sdk.stat.db.base.DbManager.BaseJob
    public String description() {
        return this.mDescription;
    }
}

File: legacy/src/main/java/com/xiaomi/mipush/sdk/stat/db/MessageDeleteJob.kt
package com.xiaomi.mipush.sdk.stat.db

import com.xiaomi.mipush.sdk.stat.db.base.DbManager

/*
 * Current override reference: com.xiaomi.xmsf 0.3.17-20260410000745 (versionCode 1003003000),
 * base.apk sha256 f3d72b6f5e1427ceecd3147a051d58e4dc95bb528397d486658e01cad9f7e590,
 * JADX path: com.xiaomi.xmsf/current/base/sources/com/xiaomi/mipush/sdk/stat/db/MessageDeleteJob.java
 * No stock 7.4.67-C same-path stat source was found in the split source tree.
 */
open class MessageDeleteJob(
    str: String,
    str2: String,
    strArr: Array<String>,
    var mDescription: String,
) : DbManager.DeleteJob(str, str2, strArr) {

    companion object {
        fun deleteUploadedJob(str: String): MessageDeleteJob {
            return MessageDeleteJob(
                str,
                "status = ?",
                arrayOf("2"),
                "a job build to delete uploaded job",
            )
        }
    }

    override fun description(): String = mDescription
}
